lib/bootstrap/2.disk.coffee

```diff
@@ -1,23 +1,21 @@
 
-module.exports = header: "Disk", handler: ({options}) ->
+module.exports = handler: ({config}) ->
```

Comment:
don't destruct the existing, header shall stay here

Reply:
Ok